project-base/src/Shopsys/ShopBundle/DataFixtures/Demo/TopProductDataFixture.php

```diff
@@ -17,39 +17,58 @@ class TopProductDataFixture extends AbstractReferenceFixture implements Dependen
      */
     protected $topProductFacade;
 
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Domain\Domain
+     */
+    protected $domain;
+
     /**
      * @param \Shopsys\FrameworkBundle\Model\Product\TopProduct\TopProductFacade $topProductFacade
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
      */
-    public function __construct(TopProductFacade $topProductFacade)
+    public function __construct(TopProductFacade $topProductFacade, Domain $domain)
     {
         $this->topProductFacade = $topProductFacade;
+        $this->domain = $domain;
     }
 
     /**
      * @param \Doctrine\Common\Persistence\ObjectManager $manager
      */
     public function load(ObjectManager $manager)
     {
-        $topProductReferenceNames = [
+        $topProductReferenceNamesOnAllDomainsExcludingSecond = [
             ProductDataFixture::PRODUCT_PREFIX . '1',
             ProductDataFixture::PRODUCT_PREFIX . '17',
             ProductDataFixture::PRODUCT_PREFIX . '9',
         ];
-
-        $this->createTopProducts($topProductReferenceNames);
+        $topProductReferenceNamesOnSecondDomain = [
+            ProductDataFixture::PRODUCT_PREFIX . '14',
+            ProductDataFixture::PRODUCT_PREFIX . '10',
+            ProductDataFixture::PRODUCT_PREFIX . '7',
+        ];
+        foreach ($this->domain->getAll() as $domainConfig) {
+            $domainId = $domainConfig->getId();
+            if ($domainId !== 2) {
```

Comment:
can you please switch the condition so it would be `$domainId === 2` ?
It just makes more sense to me :D

Reply:
fixed in dc60b55e4